core/block_svg.js

```diff
@@ -619,9 +619,22 @@ Blockly.BlockSvg.prototype.tab = function(start, forward) {
   var target = list[forward ? i + 1 : i - 1];
```

Comment:
This tab function broadly needs cleanup, but I've recorded that in Blockly: https://github.com/google/blockly/issues/1249